Library/Homebrew/language/python.rb

```diff
@@ -96,5 +96,135 @@ def self.setup_install_args(prefix)
     def self.package_available?(python, module_name)
       quiet_system python, "-c", "import #{module_name}"
     end
-  end
-end
+
+    # Mixin module for {Formula} adding virtualenv support features.
+    module Virtualenv
+      def self.included(base)
+        base.class_eval do
+          resource "homebrew-virtualenv" do
+            url "https://files.pythonhosted.org/packages/" \
+                "5c/79/5dae7494b9f5ed061cff9a8ab8d6e1f02db352f3facf907d9eb614fb80e9/" \
+                "virtualenv-15.0.2.tar.gz"
+            sha256 "fab40f32d9ad298fba04a260f3073505a16d52539a84843cf8c8369d4fd17167"
```

Comment:
Split these out as constants into a separate file; let me know if you prefer that approach.